helpers/playlistHelper.js

```diff
@@ -149,6 +148,53 @@ function getTracks(userId, playlistId, req) {
   });
 }
 
+var getEmptyPlaylist = function(accessToken, userId, playlistName, playListArr, refreshToken) {
```

Comment:
This looks like a pretty lengthy function, I'm not sure if any of this can be abstracted or not - just a thought